build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
         classpath "org.grails.plugins:hibernate5:${gormVersion-".RELEASE"}"
-        classpath "org.grails.plugins:views-gradle:1.2.9"
+        classpath 'org.grails.plugins:views-gradle:1.1.6'
```

Comment:
Do we need this view plugin at all?

Reply:
you're right, it was not needed, I removed some more unnecessary dependencies, that were automatically added, when creating the app.